platform/macos/CHANGELOG.md

```diff
@@ -15,6 +15,11 @@
 
 * Fixed a memory leak that occurred when creating a map snapshot. ([#10585](https://github.com/mapbox/mapbox-gl-native/pull/10585))
 
+## v0.6.1
```

Comment:
Guess I need to release v0.6.1 now. 😊 I can follow up with a separate merge PR that contains the tag and podspec changes for that release.